test/Spago/Build/Polyrepo.purs

```diff
@@ -0,0 +1,349 @@
+module Test.Spago.Build.Polyrepo where
+
+import Test.Prelude
+
+import Data.Array as Array
+import Data.String (Pattern(..))
+import Data.String as String
+import Node.FS.Aff as FSA
+import Node.Path as Path
+import Registry.Version as Version
+import Spago.Command.Init (DefaultConfigOptions(..))
+import Spago.Command.Init as Init
+import Spago.Core.Config as Config
+import Spago.FS as FS
+import Test.Spec (SpecT)
+import Test.Spec as Spec
+import Test.Spec.Assertions as Assert
+
+spec :: SpecT Aff TestDirs Identity Unit
+spec = Spec.describe "polyrepo" do
+
+  let
+    spagoInitCleanupNonPackageFiles spago = do
+      spago [ "init" ] >>= shouldBeSuccess
+      FSA.unlink $ Path.concat [ "src", "Main.purs" ]
+      FSA.unlink $ Path.concat [ "test", "Test", "Main.purs" ]
+      FSA.unlink "spago.yaml"
+      FS.writeYamlFile Config.configCodec "spago.yaml"
+        $ Init.defaultConfig'
+        $ WorkspaceOnly { setVersion: Just $ unsafeFromRight $ Version.parse "0.0.1" }
```

Comment:
I think this could just be this?

```suggestion
    spagoInitConfig spago = do
      FS.writeYamlFile Config.configCodec "spago.yaml"
        $ Init.defaultConfig'
        $ WorkspaceOnly { setVersion: Just $ unsafeFromRight $ Version.parse "0.0.1" }
```

Reply:
I don't understand - `spago init` boils down to "make a Main, a Test.Main, and a basic spago.yaml". (there are a few more details, but they are negligible)

In the snippet above you call `spago init`, delete the three main things it creates, and then make a new configuration.
Hence why I'm saying: we might as well just make the configuration without calling the `init` first?

Reply:
Ah, I see your point now.

Reply:
Done.